tests/src/Functional/PersonContentRenderTest.php

```diff
@@ -0,0 +1,296 @@
+    );
+    $this->assertEquals(
+      'Germany',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(3)')->text()
+    );
+
+    // Assert optional fields are not present.
+    // Assert no image is present.
+    $this->assertCount(0, $content_banner->filter('img'));
+
+    // Assert short description is not present.
+    $this->assertCount(0, $content_banner->filter('p'));
+
+    // Assert no social media links are present.
+    $this->assertCount(0, $content_banner->filter('a.standalone'));
+
+    // Assert additional information is not present.
+    $this->assertStringNotContainsString(
+      'Additional information',
+      $crawler->text()
+    );
+
+    // Assert documents are not present.
+    $this->assertStringNotContainsString(
+      'Related documents',
+      $crawler->text()
+    );
+
+    // Test person with all fields.
+    $node_with_all_fields = $this->createExamplePersonWithAllFields();
+    $this->drupalGet($node_with_all_fields->toUrl());
+    $crawler = $client->getCrawler();
+
+    $content_banner = $crawler->filter('.bcl-content-banner');
+
+    $this->assertEquals(
+      'Stefan Mayer',
+      $content_banner->filter('.card-title')->text()
+    );
+    $this->assertEquals(
+      'DG Test',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(1)')->text()
+    );
+    $this->assertEquals(
+      'Director',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(2)')->text()
+    );
+    $this->assertEquals(
+      'Germany',
+      $content_banner->filter('.card-body > div.my-4 > span:nth-child(3)')->text()
+    );
+
+    // Assert content banner image.
+    $image = $content_banner->filter('img.card-img-top');
+    $this->assertCount(1, $image);
+    $this->assertStringContainsString(
+      'image-test.png',
+      $image->attr('src')
+    );
+    $this->assertEquals('Starter Image test alt',
+      $image->attr('alt')
+    );
+
+    $this->assertEquals(
+      'This field is used to add a short biography of the person.',
+      $content_banner->filter('p')->text()
+    );
+
+    $this->assertEquals(
+      'Twitter profile',
+      $content_banner->filter('div.mb-2 > div.mb-3 > a.standalone')->text()
+    );
+
+    $content = $crawler->filter('div.col-12');
+
+    $this->assertEquals(
+      'Additional information',
+      $content->filter('h2')->eq(0)->text()
+    );
+    $this->assertEquals(
+      'Additional information example field.',
+      $content->filter('div > p')->text()
+    );
+
+    $this->assertEquals(
+      'Related documents',
+      $content->filter('h2')->eq(1)->text()
+    );
+
+    $document_group_title = $content->filter('h3.fs-4');
+    $this->assertEquals('Curriculum Vitae', $document_group_title->text());
+    $files = $content->filter('.bcl-file');
+    $this->assertEquals(
+      'Person document test',
+      $files->filter('p')->text()
+    );
+
+    $this->assertCount(3, $files);
```

Comment:
The assertions for document group don't go very deep. But maybe it's enough.
Let's keep it unless @drishu complains in next review.

Reply:
we should be using https://github.com/openeuropa/oe_bootstrap_theme/blob/1.x/tests/src/PatternAssertion/FilePatternAssert.php